From 0b265f62a9a04d5abacc86a5ebed6ddd1f063107 Mon Sep 17 00:00:00 2001
From: Juan Nunez-Iglesias <jni@fastmail.com>
Date: Tue, 23 May 2023 11:14:18 +1000
Subject: [PATCH] [Auto] Constraints upgrades: ipykernel, lit, setuptools,
 xarray (#5857)

This PR is automatically created and updated by this napari GitHub
Action:

https://github.com/napari/napari/tree/main/.github/workflows/upgrade_test_contraints.yml
It ensures that dependencies are tested with the most recent version.

The updated packages are:

 * ipykernel
 * lit
 * setuptools
 * xarray

Co-authored-by: alisterburt <alisterburt@users.noreply.github.com>
---
 resources/constraints/constraints_py3.10.txt     | 16 ++++++++--------
 resources/constraints/constraints_py3.8.txt      | 16 ++++++++--------
 resources/constraints/constraints_py3.9.txt      | 16 ++++++++--------
 .../constraints/constraints_py3.9_examples.txt   | 13 ++++++-------
 4 files changed, 30 insertions(+), 31 deletions(-)

diff --git a/resources/constraints/constraints_py3.10.txt b/resources/constraints/constraints_py3.10.txt
index d0c2cd6b..758c681d 100644
--- a/resources/constraints/constraints_py3.10.txt
+++ b/resources/constraints/constraints_py3.10.txt
@@ -52,7 +52,7 @@ coverage==7.2.5
     # via pytest-cov
 cycler==0.11.0
     # via matplotlib
-dask==2023.6.0
+dask==2023.5.0
     # via napari (setup.cfg)
 debugpy==1.6.7
     # via ipykernel
@@ -79,7 +79,7 @@ filelock==3.12.0
     #   torch
     #   triton
     #   virtualenv
-fonttools==4.39.3
+fonttools==4.39.4
     # via matplotlib
 freetype-py==2.4.0
     # via vispy
@@ -91,7 +91,7 @@ heapdict==1.0.1
     # via cachey
 hsluv==5.0.3
     # via vispy
-hypothesis==6.79.1
+hypothesis==6.75.3
     # via napari (setup.cfg)
 idna==3.4
     # via requests
@@ -108,7 +108,7 @@ in-n-out==0.1.7
     # via app-model
 iniconfig==2.0.0
     # via pytest
-ipykernel==6.22.0
+ipykernel==6.23.1
     # via
     #   napari-console
     #   qtconsole
@@ -268,7 +268,7 @@ pillow==9.5.0
     #   scikit-image
 pint==0.21
     # via napari (setup.cfg)
-platformdirs==3.7.0
+platformdirs==3.5.1
     # via
     #   jupyter-core
     #   pooch
@@ -444,7 +444,7 @@ superqt==0.4.1
     # via
     #   magicgui
     #   napari (setup.cfg)
-sympy==1.11.1
+sympy==1.12
     # via torch
 tensorstore==0.1.36
     # via napari (setup.cfg)
@@ -463,11 +463,11 @@ toolz==0.12.0
     #   dask
     #   napari (setup.cfg)
     #   partd
-torch==2.0.0
+torch==2.0.1
     # via
     #   napari (setup.cfg)
     #   triton
-tornado==6.3.1
+tornado==6.3.2
     # via
     #   ipykernel
     #   jupyter-client
diff --git a/resources/constraints/constraints_py3.8.txt b/resources/constraints/constraints_py3.8.txt
index a12ae663..884ca141 100644
--- a/resources/constraints/constraints_py3.8.txt
+++ b/resources/constraints/constraints_py3.8.txt
@@ -52,7 +52,7 @@ coverage==7.2.5
     # via pytest-cov
 cycler==0.11.0
     # via matplotlib
-dask==2023.4.1
+dask==2023.5.0
     # via napari (setup.cfg)
 debugpy==1.6.7
     # via ipykernel
@@ -79,7 +79,7 @@ filelock==3.12.0
     #   torch
     #   triton
     #   virtualenv
-fonttools==4.39.3
+fonttools==4.39.4
     # via matplotlib
 freetype-py==2.4.0
     # via vispy
@@ -91,7 +91,7 @@ heapdict==1.0.1
     # via cachey
 hsluv==5.0.3
     # via vispy
-hypothesis==6.79.1
+hypothesis==6.75.3
     # via napari (setup.cfg)
 idna==3.4
     # via requests
@@ -115,7 +115,7 @@ in-n-out==0.1.7
     # via app-model
 iniconfig==2.0.0
     # via pytest
-ipykernel==6.22.0
+ipykernel==6.23.1
     # via
     #   napari-console
     #   qtconsole
@@ -277,7 +277,7 @@ pint==0.21
     # via napari (setup.cfg)
 pkgutil-resolve-name==1.3.10
     # via jsonschema
-platformdirs==3.7.0
+platformdirs==3.5.1
     # via
     #   jupyter-core
     #   pooch
@@ -455,7 +455,7 @@ superqt==0.4.1
     # via
     #   magicgui
     #   napari (setup.cfg)
-sympy==1.11.1
+sympy==1.12
     # via torch
 tensorstore==0.1.36
     # via napari (setup.cfg)
@@ -474,11 +474,11 @@ toolz==0.12.0
     #   dask
     #   napari (setup.cfg)
     #   partd
-torch==2.0.0
+torch==2.0.1
     # via
     #   napari (setup.cfg)
     #   triton
-tornado==6.3.1
+tornado==6.3.2
     # via
     #   ipykernel
     #   jupyter-client
diff --git a/resources/constraints/constraints_py3.9.txt b/resources/constraints/constraints_py3.9.txt
index e32398cc..ffacc9bf 100644
--- a/resources/constraints/constraints_py3.9.txt
+++ b/resources/constraints/constraints_py3.9.txt
@@ -52,7 +52,7 @@ coverage==7.2.5
     # via pytest-cov
 cycler==0.11.0
     # via matplotlib
-dask==2023.6.0
+dask==2023.5.0
     # via napari (setup.cfg)
 debugpy==1.6.7
     # via ipykernel
@@ -79,7 +79,7 @@ filelock==3.12.0
     #   torch
     #   triton
     #   virtualenv
-fonttools==4.39.3
+fonttools==4.39.4
     # via matplotlib
 freetype-py==2.4.0
     # via vispy
@@ -91,7 +91,7 @@ heapdict==1.0.1
     # via cachey
 hsluv==5.0.3
     # via vispy
-hypothesis==6.79.1
+hypothesis==6.75.3
     # via napari (setup.cfg)
 idna==3.4
     # via requests
@@ -113,7 +113,7 @@ in-n-out==0.1.7
     # via app-model
 iniconfig==2.0.0
     # via pytest
-ipykernel==6.22.0
+ipykernel==6.23.1
     # via
     #   napari-console
     #   qtconsole
@@ -273,7 +273,7 @@ pillow==9.5.0
     #   scikit-image
 pint==0.21
     # via napari (setup.cfg)
-platformdirs==3.7.0
+platformdirs==3.5.1
     # via
     #   jupyter-core
     #   pooch
@@ -449,7 +449,7 @@ superqt==0.4.1
     # via
     #   magicgui
     #   napari (setup.cfg)
-sympy==1.11.1
+sympy==1.12
     # via torch
 tensorstore==0.1.36
     # via napari (setup.cfg)
@@ -468,11 +468,11 @@ toolz==0.12.0
     #   dask
     #   napari (setup.cfg)
     #   partd
-torch==2.0.0
+torch==2.0.1
     # via
     #   napari (setup.cfg)
     #   triton
-tornado==6.3.1
+tornado==6.3.2
     # via
     #   ipykernel
     #   jupyter-client
diff --git a/resources/constraints/constraints_py3.9_examples.txt b/resources/constraints/constraints_py3.9_examples.txt
index 1349dee6..1639bef1 100644
--- a/resources/constraints/constraints_py3.9_examples.txt
+++ b/resources/constraints/constraints_py3.9_examples.txt
@@ -52,7 +52,7 @@ coverage==7.2.5
     # via pytest-cov
 cycler==0.11.0
     # via matplotlib
-dask==2023.6.0
+dask==2023.5.0
     # via napari (setup.cfg)
 debugpy==1.6.7
     # via ipykernel
@@ -79,7 +79,7 @@ filelock==3.12.0
     #   torch
     #   triton
     #   virtualenv
-fonttools==4.40.0
+fonttools==4.39.4
     # via matplotlib
 freetype-py==2.4.0
     # via vispy
@@ -91,7 +91,7 @@ heapdict==1.0.1
     # via cachey
 hsluv==5.0.3
     # via vispy
-hypothesis==6.79.1
+hypothesis==6.75.3
     # via napari (setup.cfg)
 idna==3.4
     # via requests
@@ -291,7 +291,7 @@ pillow==9.5.0
     #   scikit-image
 pint==0.21
     # via napari (setup.cfg)
-platformdirs==3.7.0
+platformdirs==3.5.1
     # via
     #   jupyter-core
     #   pooch
@@ -472,7 +472,6 @@ superqt==0.4.1
     # via
     #   magicgui
     #   napari (setup.cfg)
-    #   napari-plugin-manager
 sympy==1.12
     # via torch
 tensorstore==0.1.36
@@ -494,11 +493,11 @@ toolz==0.12.0
     #   dask
     #   napari (setup.cfg)
     #   partd
-torch==2.0.0
+torch==2.0.1
     # via
     #   napari (setup.cfg)
     #   triton
-tornado==6.3.1
+tornado==6.3.2
     # via
     #   ipykernel
     #   jupyter-client
-- 
2.40.0.windows.1
